src/index.test.ts

```diff
@@ -832,6 +845,21 @@ describe('allowErrorProps(...) (#91)', () => {
 
     expect(errorAfterTransition.map).toBeInstanceOf(Map);
   });
+
+  it('custom error works with custom props', () => {
+    SuperJSON.registerClass(CustomError);
+
+    SuperJSON.allowErrorProps('rule');
```

Comment:
Related question: How do you feel about setting these for custom errors? It feels kind of redundant. Maybe we could change it so that `allowErrorProps` only affects standard errors, and for custom errors all properties are transmitted 🤔 WDYT?